blockchain/projects_test.go

```diff
@@ -512,8 +402,9 @@ func Test_TransactionExecution(t *testing.T) {
 
 }
 
+// TODO: Add account creation to playground v2
```

Comment:
It's better to avoid leaving commented-out code, I feel this is something that can be deleted and we will implement tests once we have implementation.

Reply:
Removed.